### PR TITLE
Accessibility improvements

### PR DESCRIPTION
**Pull request by jameshighfieldbjss**
Improvements based on accessibility report

- Hidden header for languages at bottom of page
- Made header a `<h3>` element (prev `<p>`) for languages as per zeplin design
- Improved header order on summary page
- Set lang attribute to html element
- Linked to input when error
- Made headings smaller above links

### CODE REVIEW

File: src/server/controllers/receipt.controller.js [hunk 1]
```diff
@@ -53,7 +53,7 @@ export const singlePaymentReceipt = async (req, res) => {
   const paymentCode = req.params.payment_code;
   try {
     const penalty = await penaltyService.getByPaymentCode(paymentCode);
-    const paymentId = `${penalty.formattedReference}_${penalty.type}`;
```
Comment (johnmorleybjss) [inline src/server/controllers/receipt.controller.js]: Does this change play nicely with all penalty types? If I recall, `formattedReference` is required to properly display IM references.

File: src/server/views/payment/index.njk [hunk 2]
```diff
@@ -18,7 +18,13 @@
             <h2 class="heading-medium error-summary-heading" id="error-summary-payment-code">
               {{ t.payment_code_page.error.title }}
             </h2>
-             {{ t.payment_code_page.error.description }}
+            <ul class="error-summary-list">
+              <li>
+                <a href="#payment_code">
```
Comment (johnmorleybjss) [inline src/server/views/payment/index.njk]: What's the goal of this link? Is it some nunjucks syntax I'm not familiar with? Or do you really want to do a hash navigation?

File: src/server/controllers/receipt.controller.js (hunk 1 above)
Comment (jameshighfieldbjss) [reply src/server/controllers/receipt.controller.js]: I incorrectly used `formattedReference` when adding the single penalty receipt which formed an incorrect payment id for immobilisations. `paymentId` is only used to retrieve the payment data from the payment service.

File: src/server/views/payment/index.njk (hunk 2 above)
Comment (jameshighfieldbjss) [reply src/server/views/payment/index.njk]: It links to the payment code input as per the examples here: https://design-system.service.gov.uk/components/error-summary/

Comment (johnmorleybjss) [reply src/server/views/payment/index.njk]: Hmm, not sure I fully understand that mechanism even after looking at those examples, but fine as long as it works!

**Pull request merged**